mhkit/wave/io/ndbc.py

```diff
@@ -207,7 +207,12 @@ def available_data(
     cache_dir = os.path.join(os.path.expanduser("~"), ".cache", "mhkit", "ndbc")
 
     # Check the cache before making the request
-    data, _, _ = handle_caching(hash_params, cache_dir, clear_cache_file=clear_cache)
+    data, _, _ = handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": None},
+        clear_cache_file=clear_cache,
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args